public/manifest.json

```diff
@@ -1,15 +1,21 @@
 {
-  "short_name": "React App",
-  "name": "Create React App Sample",
+  "name": "Nodix",
```

Comment:
Podriamos ponerle a esta tipo Nodix PWA o algo asi mas largo de nombre. Y dejamos al nombre corto solamente como Nodix, que te parece?

Reply:
hmm creo que esos nombres aparecen en la app de alguna forma, ya sea en el icono de la app o en el splash screen. Me parece que ponerle PWA quedaría medio feo :scream:

Reply:
Umm averiguemos donde aparece y en base a eso veamos si Nodix solo queda bien, no?

Reply:
Concuerdo con que PWA no me gusta en el nombre largo.
```
The short name has a maximum length of 12 characters, and is used underneath your app when it is installed on a user's homescreen.

The full name has a maximum length of 45 characters, and is used in the Chrome web store, and in the installation dialog boxes, when a user is installing your application.
```
[sauce](https://egghead.io/lessons/react-change-the-name-and-short-name-of-a-pwa-built-with-create-react-app)

Reply:
Listo entonces! Excelente =)

:hotdog: